backend/src/main/java/com/karankumar/bookproject/backend/model/bookImport/libraryThing/LibraryThingBookImport.java

```diff
@@ -0,0 +1,28 @@
+package com.karankumar.bookproject.backend.model.bookImport.libraryThing;
+
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class LibraryThingBookImport {
+
+  @JsonProperty("title")
+  private String title;
+  
+  @JsonProperty("primaryauthor")
```

Comment:
Could you make this camelCase (primaryAuthor)?

Reply:
Books export data from Library Things has this property as a non-cammel case (`primaryauthor`). If I change that it will be not parsable